tasks/riscv64-port-plan.md

```
@@ -0,0 +1,298 @@
+executed:
+
+```
+Unable to resolve @typescript/native-preview-linux-riscv64. Either your platform is
+unsupported, or you are missing the package on disk.
+```
+
+So `npm ci` looks green and the build is the thing that breaks. A CI job that gates on
+install success would pass and mislead. (Install itself took 3m for one package, versus
+41s for pi's whole 127-package tree — npm spent that time resolving, not compiling.)
+
+#### Phase 1 finding: `tsc` is NOT a drop-in for `tsgo`
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)